src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -40,6 +40,14 @@
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.Builder.cs" />
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.Enumerator.cs" />
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.CollectionInterfaces.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.IImmutableList.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.IndexOf.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.Insert.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.OfType.cs" />
```

Comment:
This file contains a single method that is less than 10 LOC, so I'm not so sure it deserves its own file. Maybe move this back into `ImmutableArray_1.cs`